src/starknet-wrappers.ts

```diff
@@ -173,51 +152,6 @@ export abstract class StarknetWrapper {
 
     public abstract compileSierraToCasm(options: SierraToCasmOptions): Promise<ProcessResult>;
```

Comment:
Is compilation the only reason why we need the StarknetWrapper logic?

I'll just mention this here: https://github.com/0xSpaceShard/starknet-hardhat-plugin/issues/387
So that the issue has a link to this comment, we don't have to deal with the issue here.

Reply:
`starknet.js` could also be used to replace the hashing utilities although I believe it would be less performant.